tests/usecase.py

```diff
@@ -6,19 +6,25 @@
 
 import subprocess
 import time
+import os
 
+# Directory for cloning Apache Kafka sources to.
+# It will only be used if 'version' is not specified.
+# Otherwise Apache Kafka binary distrubution will be downloaded to
+# a cluster data dir ( trivup/tmp/TestCluster/KafkaBrokerApp/kafka/conf['version'] )
+kafka_dir=os.path.join(os.path.expanduser('~'), "src", "kafka")
 
 if __name__ == '__main__':
     cluster = Cluster('TestCluster', 'tmp')
 
     # One ZK
-    zk1 = ZookeeperApp(cluster, bin_path='/home/maglun/src/kafka/bin/zookeeper-server-start.sh')
+    zk1 = ZookeeperApp(cluster, bin_path=os.path.join(kafka_dir, '/bin/zookeeper-server-start.sh'))
```

Comment:
I think we can safely remove bin_path here too.

in fact, I don't think the use-case needs to show off local source dir at all, so you can remove kafka_dir up top too.